README.rst

```diff
@@ -25,23 +25,30 @@ PennyLane Qiskit Plugin
     :alt: PyPI - Python Version
     :target: https://pypi.org/project/PennyLane-qiskit
 
+
```

Comment:
Would be cool to go from `PennyLane Qiskit` to `PennyLane-Qiskit` on lines 1 and 71 (leaving it here for lack of a better place for the suggestion :sweat_smile: ).